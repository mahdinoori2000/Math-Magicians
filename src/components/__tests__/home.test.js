import { render } from '@testing-library/react';
import Home from '../home';

test('should render the home section correctly', () => {
  const tree = render(<Home />);
  expect(tree).toMatchSnapshot();
});
