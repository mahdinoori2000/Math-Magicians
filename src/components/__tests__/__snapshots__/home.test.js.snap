// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home section correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <h1>
          Welcome to Math Magicians!
        </h1>
        <p>
          Math Magician is a sleek and intuitive website featuring a powerful calculator for mathematical operations. With its simple and elegant design, performing calculations becomes a breeze.
        </p>
        <p>
          Additionally, the website offers an inspiring Quote section, where users can discover motivational quotes fetched from an external API. This single-page application combines functionality and aesthetics, providing an enjoyable user experience.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <h1>
        Welcome to Math Magicians!
      </h1>
      <p>
        Math Magician is a sleek and intuitive website featuring a powerful calculator for mathematical operations. With its simple and elegant design, performing calculations becomes a breeze.
      </p>
      <p>
        Additionally, the website offers an inspiring Quote section, where users can discover motivational quotes fetched from an external API. This single-page application combines functionality and aesthetics, providing an enjoyable user experience.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
